data/presets/man_made/carpet_hanger

```diff
@@ -0,0 +1,21 @@
+{
+    "icon": "maki-picnic-site",
+    "fields": [
+        "material",
+        "height",
+        "colour"
+    ],
+    "geometry": [
+        "point",
+        "line"
```

Comment:
According to the wiki it's only [intended](https://wiki.openstreetmap.org/wiki/Tag:man_made%3Dcarpet_hanger) to be used on nodes. Currently, there are only 14 mapped as ways (compared to 574 as nodes). But I see the rationale and similarity to how [`amenity=bench`](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dbench) was later [extended](https://wiki.openstreetmap.org/w/index.php?title=Tag:amenity%3Dbench&diff=prev&oldid=705902) to linear features as well.

My request would be to update this on the wiki fist.

Reply:
Thanks for the tip, I've just updated the wiki.